blog/2025-03-18-vercel-flags-sdk.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: "vercel-flags-sdk-adapter"
+title: "Vercel Flags SDK - OpenFeature Adapter"
+date: 2025-03-18
+authors: [jonathannorris]
+description: "Announcing Vercel Flags SDK OpenFeature Adapter"
+tags: [vercel, "flags sdk", "next.js", "openfeature adapter"]
+draft: false
+---
+
+We are happy to share the news that Vercel's new open-source [Flags SDK](https://flags-sdk.dev/) for Next.js and SvelteKit has shipped with a native [OpenFeature Adapter](https://flags-sdk.dev/docs/api-reference/adapters/openfeature).
+The Flags SDK is an exciting development for standardizing feature flagging in Next.js and SvelteKit, as it takes an [oppinionated approach](https://vercel.com/blog/flags-as-code-in-next-js) to how feature flags should be used in Next.js:
```

Comment:
There are a few mentions of "nexjs and svelte kit" in the doc. This use seems close to the previous one and maybe we could use a slightly different phrasing?